starport/services/chain/serve.go

```diff
@@ -299,6 +303,28 @@ func (c *Chain) serve(ctx context.Context, forceReset bool) error {
 		return err
 	}
 
+	// we also consider the binary in the checksum to ensure the binary has not been changed by a third party
+	binaryModified := false
+	binaryName, err := c.Binary()
+	if err != nil {
+		return err
+	}
+	binaryPath, err := exec.LookPath(binaryName)
+	if err != nil {
+		if strings.Contains(err.Error(), "file not found") {
```

Comment:
https://golang.org/pkg/os/exec/#pkg-variables

Reply:
Oh I missed that